spreadd/src/components/CreatorStore/CreatorStore.jsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+
+import {
+  StyledSection,
+  StyledContainer,
+  StyledSvgContainer,
+  StyledWrapper,
+  StyledLeftWrapper,
+  StyledCardContainer,
+  StyledCardHeading,
+  StyledCardText,
+  StyledImageContainer,
+  StyledPhoneImage,
+  StyledRightWrapper,
+  StyledCreatorStore,
+  StyledRightInner,
+  StyledHeading,
+  StyledParagraph,
+  // StyledCreatorCard, {Pink line styling component}
+} from "./CreatorStore.styled";
+
+import image from "../../assets/rightdot.svg";
+
+export const CreatorStore = () => {
+  return (
+    <div>
+      <StyledSection>
+        <StyledContainer>
+          <StyledSvgContainer>
+            <img src={image} alt="rightdot" />
+          </StyledSvgContainer>
+
+          <StyledWrapper>
+            {/* Left section is going to start */}
```

Comment:
dont need to give this type of comments if that is big section than only you give @KBhautik6